test/corpus/expressions.txt

```diff
@@ -825,15 +836,14 @@ switch { open Mod; foo() } {
           (call_expression (value_identifier) (arguments))))
       (switch_match
         (value_identifier)
-        (expression_statement (number))))))
+        (sequence_expression (expression_statement (number)))))))
 
 ===========================================
 Switch parenthesized
 ===========================================
 
 switch n {
 | (1 | 2) => 1
-| (n if n > 10 | n if n < 42) | 50 | (n if n > 100) => n
```

Comment:
This match case is not valid, see [Playground](https://rescript-lang.org/try?code=M4dwlgLgxgFgBAOzgbwFAB84AoCMdMBMAlHALwB8cOG2SYAZonJTgAz5MNMA8cALMQ4BWdpix1GSFq1YkKiGgH0ylALTUAvkA)

The `if` or `when` condition cannot occur inside `parenthesized_pattern`.